### DISCUSSION TITLE
Header examples (v0.100)

### DISCUSSION THREAD

**Discussion by hochstibe** [fastapi/fastapi#9815] [Questions] [answered]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/pydantic/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
from typing import Annotated

from fastapi import FastAPI, Header, Body


app = FastAPI()


@app.get("/")
def header(
    header_param: Annotated[
        str | None,
        Header(examples=["not working"]),
    ] = None,
):
    return {"header": header_param}


@app.post("/")
def body(
    body_param: Annotated[
        dict | None,
        Body(examples=[{"working": "example"}]),
    ] = None,
):
    return {"body": body_param}
```

### Description

Before, I used the `example` attribute, in Version 0.99 it is marked as deprecated. 

The documentation has a (working) example of a body parameter: [body with examples](https://fastapi.tiangolo.com/tutorial/schema-extra-example/#body-with-examples)

When I set the example in the header parameter as `examples=["not working"]`, it is not set in the openapi documentation.
For the body parameter, it works well (second endpoint)

How should I declare header parameters?

### Operating System

Windows

### FastAPI Version

0.100.0-beta-3

### Python Version

3.10.2

**Comment by mglasner** [comment]
I am having a very similar problem with multiple examples in Body, only the first example appear in UI and the old dropdown manu to select examples is gone.

**Reply by jgould22** [reply, marked as answer]
You may find some explanation for that in this thread https://github.com/tiangolo/fastapi/discussions/9794

**Reply by hochstibe** [reply]
It explains the reasons for changing to examples and is related. I continued in the other thread...

**Reply by tiangolo** [reply]
There's a new parameter `openapi_examples` just for this, to allow declaring examples that show up in Swagger UI while it implements support for multiple JSON Schema examples. It's available now in FastAPI 0.103.0, the new docs are here: https://fastapi.tiangolo.com/tutorial/schema-extra-example/#openapi-specific-examples 🎉

**Comment by hochstibe** [comment]
The Header-examples are added to the `openapi.json`, therefore the bug seems to be in OpenAPI (or another format is required, Im not familiar with the specification)

**Comment by MrCreosote** [comment]
https://github.com/swagger-api/swagger-ui/issues/9045
